app/dashboards/unit_dashboard.rb

```diff
@@ -1,4 +1,5 @@
 require "administrate/base_dashboard"
+require "page_link_field"
```

Comment:
I don't think you should need these requires here, I would have thought it would have been auto loaded?

Reply:
Yes, not required and removed.